docs/application/native/guides/ui/dali/capture.md

```diff
@@ -0,0 +1,67 @@
+# Capture
+
+
+Capture snapshot the current DALi scene. The captured result can be saved as a file and also can be used to be rendered by DALi `ImageView` or `ImageVisual`.
+
+In this tutorial, we show how to capture scene and to use it.
+
+
+## Tutorial
```

Comment:
Please remove this heading or add a different sub-heading.

Reply:
Thank you I changed it